stdpopsim/slim_engine.py

```diff
@@ -1422,7 +1561,7 @@ def _recap_and_rescale(
         # Creating a SLiM mutation model
         # TODO: fix this up to do the right types? or, share them?
         model = msprime.SLiMMutationModel(
-            type=len(contig.mutation_types()), next_id=max_id + 1
```

Comment:
need to fix this. find which mutation type wasn't simulated by SLiM and add them here!